src/main/java/me/steinborn/krypton/mixin/shared/network/flushconsolidation/ThreadedAnvilChunkStorageMixin.java

```diff
@@ -64,60 +101,124 @@ public void updatePosition(ServerPlayerEntity player) {
                 entityTracker.updateTrackedStatus(player);
             }
         }
-
+        
         ChunkSectionPos oldPos = player.getWatchedSection();
         ChunkSectionPos newPos = ChunkSectionPos.from(player);
         boolean isWatchingWorld = this.playerChunkWatchingManager.isWatchDisabled(player);
         boolean noChunkGen = this.doesNotGenerateChunks(player);
         boolean movedSections = !oldPos.equals(newPos);
-
+        
```

Comment:
Why the whitespace changes?

Reply:
My bad, my custom formatting settings automatically add the whitespace. I'll remove it in ~30 (also, if you add a `.editorconfig` to the root directory, it'll have everyone else's code formatters match yours. Assuming you're using intellij)